### ISSUE TITLE
Make failed due to "ModuleNotFoundError" when run ./ballisticacore_server

### ISSUE THREAD

**Issue by Zee-hash** [efroemling/ballistica#37]
If I just execute `make prefab-linux-server-release-build` or `make prefab-linux-server-debug-build`, this problem will not occur. This means that this error only appears when I run `./ballisticacore_server`. Is it caused by this module not in the current directory or the path not in my environment variables? 
```
user@LapTop:~/ballistica$ make prefab-linux-server-debug
make[1]: Entering directory '/home/user/ballistica/assets'
make[2]: Entering directory '/home/user/ballistica/assets'
make[2]: Nothing to be done for 'assets-cmake'.
make[2]: Leaving directory '/home/user/ballistica/assets'
make[1]: Leaving directory '/home/user/ballistica/assets'
Traceback (most recent call last):
  File "./ballisticacore_server", line 34, in <module>
    from bacommon.servermanager import ServerConfig, StartServerModeCommand
ModuleNotFoundError: No module named 'bacommon'
Makefile:268: recipe for target 'prefab-linux-server-debug' failed
make: *** [prefab-linux-server-debug] Error 1

user@LapTop:~/ballistica$ make prefab-linux-server-release
make[1]: Entering directory '/home/user/ballistica/assets'
make[2]: Entering directory '/home/user/ballistica/assets'
make[2]: Nothing to be done for 'assets-cmake'.
make[2]: Leaving directory '/home/user/ballistica/assets'
make[1]: Leaving directory '/home/user/ballistica/assets'
Traceback (most recent call last):
  File "./ballisticacore_server", line 34, in <module>
    from bacommon.servermanager import ServerConfig, StartServerModeCommand
ModuleNotFoundError: No module named 'bacommon'
Makefile:299: recipe for target 'prefab-linux-server-release' failed
make: *** [prefab-linux-server-release] Error 1
```

**Comment by Dliwk** [comment]
As hotfix, you may run `export PYTHONPATH=dist/ba_data/python` before running `./ballisticacore_server`

**Comment by efroemling** [comment]
Oops; looks like this might have broke in a recent PR; I should have caught that. There is code in the script that adds dist/ba_data/python to the path before attempting the import, but it seems to have been moved to after the import. I'll have a fix in soon. Thanks for the heads-up.

**Comment by efroemling** [comment]
Ok this should be fixed with fab6a54ff8ceb68c2869d31647f65f8b026c2ef9
Thanks again for the heads-up.
